docs/internal/flower.md

```diff
@@ -0,0 +1,18 @@
+# Celery admin panel (internal)
+
```

Comment:
I created `internal` folder for docs, that wouldn't published on official doc site, it's just for us, for internal usage :)